src/System.Net.Http/src/netcore50/System/Net/HttpClientHandler.cs

```diff
@@ -161,11 +161,14 @@ public bool PreAuthenticate
             get { return true; }
             set
             {
+                /*
+                TODO:#18104
```

Comment:
I don't have a problem with commenting this code out now.  @tijoytom is trying to get the UWP automated test runs unblocked and the PNSE is blocking that. They are working ahead of our team which plans to address this in the longer term.

The proper fix is more complicated because the UWP HTTP stack only support PreAuthenticate=true.  There is no concept of false because it uses WinRT Windows.Web.Http which uses Wininet. Wininet always uses a cached authentication model and does preauthenticate as much as possible.

We will address this later as part of our work for UWP.